DashWallet/Sources/Models/CrowdNode/TxFilters/FullCrowdNodeSignUpTxSet.swift

```diff
@@ -60,7 +66,7 @@ final class FullCrowdNodeSignUpTxSet: TransactionWrapper {
             CrowdNodeResponse(responseCode: ApiCode.pleaseAcceptTerms, accountAddress: nil),
         ]
 
-        if let accountAddress = CrowdNodeDefaults.shared.crowdNodeAccountAddress {
```

Comment:
This was reading the storage every time `tryInclude` is run. Dumb mistake on my part.